CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 4.2.0 (Feature release)
```

Comment:
I've just looked back at previous PRs and actually, I don't think we update this until we do the release (e.g: on Monday). Sorry, totally my fault I didn't think of that earlier!

```suggestion
## Unreleased
```